packages/universal/reactive/README.md

```diff
@@ -64,3 +69,76 @@ A marker has these fundamental operations:
 > `get` for each entry in the map. This means that if a formula used a `has`
 > check to check for a key's presence and it returns `true`, updating the value
 > of that key will not invalidate the formula.
+
+### Formula
```

Comment:
what's the advantage of using a vanilla formula over just a function?

Reply:
You can subscribe to a vanilla formula directly. TL;DR even though `Formula` doesn't cache, it still has dependencies and is still a `Reactive<T>`.

If you don't need a `Reactive<T>`, just use a regular function and call it a day :)